app/models/child_object.rb

```diff
@@ -43,6 +43,7 @@ def access_master_url
   def access_master_path
     return @access_master_path if @access_master_path
     image_mount = ENV['ACCESS_MASTER_MOUNT'] || "data"
+    # TODO(alishaevn): the "pairtree_path" variable is coming back as "nil" in the batch process child detail spec
```

Comment:
JP and I paired on diagnosing this, fixing the test, and writing a bug ticket, so I don't think we still need this TODO here (see https://github.com/yalelibrary/YUL-DC/issues/827)

Reply:
Ah yes, good catch. Sorry, forgot about that comment.